eth/stagedsync/stage_execute.go

```diff
@@ -129,6 +130,11 @@ func executeBlock(
 		if err = rawdb.AppendReceipts(tx, blockNum, receipts); err != nil {
 			return err
 		}
+
+		if stateSyncReceipt != nil {
+			rawdb.WriteBorReceipt(tx, block.Hash(), block.NumberU64(), stateSyncReceipt)
```

Comment:
Can you Append borReceipt instead of Put?

Reply:
WriteBorTxLookupEntry should I use Append on this one instead of Put too?

Reply:
> WriteBorTxLookupEntry should I use Append on this one instead of Put too?

use Put where doing "random updates/inserts" and Append where "append" to end of table (it will lead to 2x smaller table size)